release-notes/opensearch-security.release-notes-2.19.0.0.md

```diff
@@ -0,0 +1,68 @@
+* add ingest pipeline and indices related permissions for anomaly_full_access role ([#5069](https://github.com/opensearch-project/security/pull/5069))
+* Added roles for ltr read and full access ([#5070](https://github.com/opensearch-project/security/pull/5070))
+
+### Bug Fixes
+* Fix issue with jwt attribute parsing of lists ([#4885](https://github.com/opensearch-project/security/pull/4885))
+* Log io.netty.internal.tcnative.SSLContext availability warning only when OpenSSL is explicitly enabled but not available ([#4906](https://github.com/opensearch-project/security/pull/4906))
+* Reduce log level in HttpJwtAuthenticator if request cannot be authenticated ([#4917](https://github.com/opensearch-project/security/pull/4917))
+* Honor log_request_body setting in compliance audit log ([#4918](https://github.com/opensearch-project/security/pull/4918))
+* Change log level for log line in OBO Authenticator if OBO is disabled ([#4956](https://github.com/opensearch-project/security/pull/4956))
+* Set default value for key/trust store type as constant for JDK PKCS setup ([#5003](https://github.com/opensearch-project/security/pull/5003))
+* Fix SSL config for JDK PKCS setup ([#5033](https://github.com/opensearch-project/security/pull/5033))
+* Fix Netty4 header verifier inbound handler to deal with upgrade requests ([#5045](https://github.com/opensearch-project/security/pull/5045))
+* Generate jacoco report for integTestRemote task ([#5050](https://github.com/opensearch-project/security/pull/5050))
+
+### Maintenance
+* Bump org.junit.jupiter:junit-jupiter-api from 5.11.2 to 5.11.3 ([#4856](https://github.com/opensearch-project/security/pull/4856))
+* Bump ch.qos.logback:logback-classic from 1.5.11 to 1.5.12 ([#4857](https://github.com/opensearch-project/security/pull/4857))
+* Bump com.google.errorprone:error_prone_annotations from 2.34.0 to 2.35.1 ([#4850](https://github.com/opensearch-project/security/pull/4850))
+* Bump org.junit.jupiter:junit-jupiter from 5.11.2 to 5.11.3 ([#4861](https://github.com/opensearch-project/security/pull/4861))
+* Bump Wandalen/wretry.action from 3.5.0 to 3.7.0 ([#4874](https://github.com/opensearch-project/security/pull/4874))
+* Bump org.checkerframework:checker-qual from 3.48.1 to 3.48.2 ([#4875](https://github.com/opensearch-project/security/pull/4875))
+* Bump com.nimbusds:nimbus-jose-jwt from 9.41.2 to 9.45 ([#4876](https://github.com/opensearch-project/security/pull/4876))
+* Bump com.nimbusds:nimbus-jose-jwt from 9.45 to 9.46 ([#4890](https://github.com/opensearch-project/security/pull/4890))
+* Bump Wandalen/wretry.action from 3.7.0 to 3.7.2 ([#4891](https://github.com/opensearch-project/security/pull/4891))
+* Bump Zookeeper to 3.9.3 ([#4895](https://github.com/opensearch-project/security/pull/4895))
+* Bump com.nimbusds:nimbus-jose-jwt from 9.46 to 9.47 ([#4916](https://github.com/opensearch-project/security/pull/4916))
```

Comment:
nonblocking nit: do we want to combine the same dependency bump into one bullet point (probably doesn't matter)?

Reply:
Yea, I agree they should be combined. I'm wondering if security should start following the CHANGELOG model present in other repositories to capture all entries in the changelog as they are merged in.